apps/obsidian/src/components/TldrawViewComponent.tsx

```diff
@@ -21,14 +41,86 @@ export const TldrawPreviewComponent = ({
     return () => clearTimeout(timer);
   }, []);
 
-  const handleMount = useCallback((editor: Editor) => {
-    editor.setCurrentTool("hand");
-    editor.updateInstanceState({});
+  const saveChanges = useCallback(async () => {
+    const newData = getTLDataTemplate({
+      pluginVersion: plugin.manifest.version,
+      tldrawFile: createRawTldrawFile(currentStore),
+      uuid: window.crypto.randomUUID(),
```

Comment:
@trangdoan982 Just noticed, since we are generating a new uuid on each `saveChanges`, `stringifiedData === lastSavedDataRef.current` will always be false.